src/hammer-vlsi/technology/asap7/defaults.yml

```diff
@@ -79,3 +79,11 @@ mentor.extra_env_vars:
   - VAR_M5_XOFFSET: "0"
   - VAR_M6_YOFFSET: "0"
   - VAR_M7_XOFFSET: "0"
+
+par.inputs:
+  # gds_merge: ture is mandatory,
+  # since asap7/__init__.py:130, the shrink script will filter stdcell module with keyword "ASAP7_75t"
+  # thus we should merge gds by PaR tools firstly,
+  # TODO: try to change export_config_outputs for PaR inputs to expose gds_merge to output json?
```

Comment:
I think we do want to try and pass it through since the reason why this is an input is because some projects may want to use a 3rd-party tool to merge the GDS

Reply:
This may suffice for now if we file an issue later